src/opnsense/service/templates/OPNsense/Proxy/+TARGETS

```diff
@@ -8,3 +8,4 @@ post-auth.conf:/usr/local/etc/squid/post-auth/dummy.conf
 pre-auth.conf:/usr/local/etc/squid/pre-auth/dummy.conf
 rc.conf.d:/etc/rc.conf.d/squid/squid
 squid.conf:/usr/local/etc/squid/squid.conf
+wpad.dat:/usr/local/www/wpad.dat
```

Comment:
It’s a bit strange to drop files into www but I know what you are after

Reply:
Do you have a better idea? The URL does not really match the pattern of standard OPNsense rules. Otherwise we would have to heavily rewrite routes in lighttpd.

Reply:
No, I’m only raising awareness. In any case this will not merge before stable/18.1 is branched so we have time to think about this. For now this is good enough for -devel

Reply:
the only thing I could think of is adding a custom rewrite to a controller without authentication.